### ISSUE TITLE
FATAL ERROR when trying to navigate inside the editor using VoiceOver

### ISSUE THREAD

**Issue by nosami** [mono/monodevelop#5887]
Issue found on master.
1. Switch VoiceOver and accessibility preference on 
2. Shift+Ctrl+Alt and arrow into editor
3. Ctrl+Alt+Arrow up and down 

```
FATAL ERROR [2018-09-07 16:50:04Z]: Visual Studio failed to start. Some of the assemblies required to run Visual Studio (for example gtk-sharp)may not be properly installed in the GAC.
System.NullReferenceException: Object reference not set to an instance of an object
 at Mono.TextEditor.TextViewMargin+TextViewMarginAccessibilityProxy.GetRangeForLine (System.Int32 lineNo) [0x00013] in /Users/jason/src/monodevelop/main/src/addins/MonoDevelop.SourceEditor2/Mono.TextEditor/Gui/TextViewMargin.cs:242
 at (wrapper delegate-invoke) System.Func`2[System.Int32,MonoDevelop.Components.AtkCocoaHelper.AtkCocoa+Range].invoke_TResult_T(int)
 at MonoDevelop.Components.AtkCocoaHelper.RealAccessibilityElementNavigableStaticTextProxy.AccessibilityRangeForLine (System.nint line) [0x00001] in /Users/jason/src/monodevelop/main/src/core/MonoDevelop.Ide/MonoDevelop.Components.AtkCocoaHelper/AtkCocoaHelperMac.cs:1572
```

> VS bug [#681237](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/681237)

**Comment by xamarin-release-manager** [comment]
Fixed in version 7.8.0.1362 (master)

Pull Request #6724 merged by: Lluis Sanchez
Author: mono
Commit: fc834ddf2e5e488fad96ea9dcc6a270abc324c0e (mono/monodevelop)

**Comment by xamarin-release-manager** [comment]
Fixed in version 8.0.0.901 (release-8.0)

Pull Request #6724 merged by: Lluis Sanchez
Author: mono
Commit: fc834ddf2e5e488fad96ea9dcc6a270abc324c0e (mono/monodevelop)

**Comment by xamarin-release-manager** [comment]
Fixed in version 8.6.0.802 (master)

Pull Request #6724 merged by: Lluis Sanchez
Author: mono
Commit: fc834ddf2e5e488fad96ea9dcc6a270abc324c0e (xamarin/monodevelop)
